docker/koordlet.dockerfile

```diff
@@ -4,14 +4,22 @@ WORKDIR /go/src/github.com/koordinator-sh/koordinator
 COPY go.mod go.mod
 COPY go.sum go.sum
 
-RUN go mod download
+RUN go env -w GO111MODULE=on && \
+    go env -w GOPROXY=https://goproxy.cn,direct && \
+    go mod download
 
 COPY apis/ apis/
 COPY cmd/ cmd/
 COPY pkg/ pkg/
 
 RUN GOOS=linux GOARCH=amd64 go build -a -o koordlet cmd/koordlet/main.go
 
+# The CUDA container images provide an easy-to-use distribution for CUDA supported platforms and architectures.
+# NVIDIA provides rich images in https://hub.docker.com/r/nvidia/cuda/tags, literally cover all kinds of CUDA version
+# and system architecture. Please replace the following base image according to your Kubernetes/System environment.
+# More details about how those images got built, you might wanna check the original Dockerfile in
+# https://gitlab.com/nvidia/container-images/cuda/-/tree/master/dist.
```

Comment:
This comment is very good, maybe we need a chapter to describe the cuda version problem when we publish it, it is really a headache.